download_logs.py

```diff
@@ -0,0 +1,71 @@
+!import os, glob
```

Comment:
@joaeba sorry just lurking here, but noticed this typo was committed by accident instead of the usual shebang